index.d.ts

```diff
@@ -360,6 +360,11 @@ interface ConfigType {
     section: string;
     sharedAdTargeting: { [key: string]: any };
     isPaidContent?: boolean;
+    pageId?: string;
+    webPublicationDate?: number;
+    headline?: string;
+    author?: string;
+    keywords?: string;
```

Comment:
I'm a little out of my depth here in terms of how commercial works so take this comment with that in mind!

So, I'm moderately sure that we already have all these values on the `CAPI` object, which is in scope at the top of this file. I know commercial tend to only use `config` but maybe we could source this data from CAPI to save on repetition?

Reply:
That said, I don't see `keywords` anywhere, is it the same as `keywordIds`?

Reply:
@oliverlloyd Like you said`CAPIType` doesn't have keywords. I'd keep using the config, since our other permutive integrations work this way (we wouldn't have any other way of doing it there I believe). Would creating a `CommercialConfig` type for our purposes and then let the `ConfigType` extend it help at all? Happy to chat IRL.

Reply:
Fair enough, and having the separation (DCR/Commercial) is actually quite nice. To be honest, I'd need to look into things some more to know if having `CommercialConfig` is a good way to go or not. I'm happy to take the simplest route here for now and revisit later. We already have a task to refactor how we pass data for DCR so it will get swept up in that.